docs/en/quick-answers/troubleshooting/houdini-integrations-not-starting.md

```diff
@@ -34,4 +34,6 @@ Then you should add $HOUDINI_PATH; to the end of the path defined in the file an
 
 This will allow the Shotgun set value to persist when Houdini launches.
 
-If this does not fix your problem please reach out to our support team and they will help you diagnose the issue.
\ No newline at end of file
+On Windows we've seen `$HOUDINI_PATH` causing issue with it to try and bootstrap multiple times. If you get this you should try using `%HOUDINI_PATH%` instead.
```

Comment:
This is kind of a weird sentence; can we rephrase?

Reply:
That is weird :D
Changed to a warning box with this content:

On Windows we've seen `$HOUDINI_PATH` causing issues. It sometimes tries to bootstrap the Shotgun integration multiple times generating an error like this: `Toolkit bootstrap is missing a required variable : TANK_CONTEXT`. If you get this you should try using `%HOUDINI_PATH%` instead.

Reply:
much better!!